standard/standard-library.md

```diff
@@ -160,11 +160,19 @@ namespace System
             Exception innerException);
     }
 
-    public class NotSupportedException : Exception
```

Comment:
I think we still need NotSupportedException for `IEnumerator.Reset`.

Reply:
I think that snuck in by accident. Reverting.